src/App.js

```diff
@@ -0,0 +1,80 @@
+import React, {useState} from "react";
+import "./App.css"
+import {
+    BrowserRouter as Router,
+    Switch,
+    Route,
+} from "react-router-dom";
+import styled from "styled-components"
+import {patientsFake} from "./patientsFake";
+import DoctorPage from "./Components/DoctorPage/DoctorPage";
+import SignPage from "./Components/LoginPage/SignPage";
+import PatientPage from "./Components/PatientPage/PatientPage";
+import {TestButtonPage} from "./Components/Test/TestButtonPage";
+
+const doctorsFake = [
+    {
```

Comment:
Please move this mock to mock.js file of smth like this

Reply:
Done. may be it not in right folder...